Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair.h

```diff
@@ -1466,7 +1466,7 @@ bool remove_degenerate_faces(      TriangleMesh& tmesh,
 }
 
 template<class TriangleMesh>
-std::size_t remove_degenerate_faces(TriangleMesh& tmesh)
+bool remove_degenerate_faces(TriangleMesh& tmesh)
```

Comment:
@sloriot @MaelRL Please review this change.

Reply:
@sloriot changed the return type in e4ad5d96a7afd1f2cca3fbc7cb2661a1c1be7659, but forgot to change the return type in the named parameter-less call.